src/errors/runtime/templates.js

```diff
@@ -7,101 +7,100 @@ import { RUNTIME_ERRORS } from '../types';
+    [RUNTIME_ERRORS.optionValueIsNotValidKeyValue]:                      'The "{optionName}" option does not contain a valid key-value pair.',
+    [RUNTIME_ERRORS.invalidQuarantineOption]:                            'The "{optionName}" option is not valid. Specify "attemptLimit" and "successThreshold" to configure quarantine mode.',
+    [RUNTIME_ERRORS.invalidQuarantineParametersRatio]:                   'The value of "attemptLimit" ({attemptLimit}) should be greater then the value of "successThreshold" ({successThreshold}).',
+    [RUNTIME_ERRORS.invalidAttemptLimitValue]:                           'The "{attemptLimit}" parameter only accepts values of {MIN_ATTEMPT_LIMIT} and up.',
+    [RUNTIME_ERRORS.invalidSuccessThresholdValue]:                       'The "{successThreshold}" parameter only accepts values of {MIN_SUCCESS_THRESHOLD} and up.',
     [RUNTIME_ERRORS.invalidSpeedValue]:                                  'Speed should be a number between 0.01 and 1.',
-    [RUNTIME_ERRORS.invalidConcurrencyFactor]:                           'The concurrency factor should be an integer greater or equal to 1.',
+    [RUNTIME_ERRORS.invalidConcurrencyFactor]:                           'The concurrency factor should be an integer greater than or equal to 1.',
     [RUNTIME_ERRORS.cannotDivideRemotesCountByConcurrency]:              'The number of remote browsers should be divisible by the factor of concurrency.',
-    [RUNTIME_ERRORS.cannotSetConcurrencyWithCDPPort]:                    'The concurrency factor should be used without CDP port.',
-    [RUNTIME_ERRORS.portsOptionRequiresTwoNumbers]:                      'The "--ports" option requires two numbers to be specified.',
-    [RUNTIME_ERRORS.portIsNotFree]:                                      'The specified {portNum} port is already in use by another program.',
-    [RUNTIME_ERRORS.invalidHostname]:                                    'The specified "{hostname}" hostname cannot be resolved to the current machine.',
-    [RUNTIME_ERRORS.cannotFindSpecifiedTestSource]:                      'Cannot find a test source file at "{path}".',
-    [RUNTIME_ERRORS.clientFunctionCodeIsNotAFunction]:                   '{#instantiationCallsiteName} code is expected to be specified as a function, but {type} was passed.',
-    [RUNTIME_ERRORS.selectorInitializedWithWrongType]:                   '{#instantiationCallsiteName} is expected to be initialized with a function, CSS selector string, another Selector, node snapshot or a Promise returned by a Selector, but {type} was passed.',
+    [RUNTIME_ERRORS.cannotSetConcurrencyWithCDPPort]:                    'The concurrency factor can not include the CDP port.',
+    [RUNTIME_ERRORS.portsOptionRequiresTwoNumbers]:                      'The "--ports" argument accepts two values at a time.',
+    [RUNTIME_ERRORS.portIsNotFree]:                                      'Port {portNum} is occupied by another process.',
```

Comment:
It's very good that you recheck all messages.
However, some messages are written in the way same as in other programs.
For example, `RUNTIME_ERRORS.portIsNotFree`.
Try to google the `The specified port is already in use by another program.` You will find that the same message is used for IIS Express, Django, and etc.

Try to google other messages and consider leaving the old format.